test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -257,14 +257,18 @@ public void setUp() throws Exception {
 
         pluginConfig = new LogReplicationPluginConfig(pluginConfigFilePath);
 
-        destMetadataManager = new LogReplicationMetadataManager(dstTestRuntime,
-                new LogReplicationContext(new LogReplicationConfigManager(dstTestRuntime, session.getSinkClusterId()), 0,
-                "test" + SERVERS.PORT_0, true, pluginConfig, dstTestRuntime));
+        LogReplicationContext destContext = new LogReplicationContext(new LogReplicationConfigManager(dstTestRuntime, session.getSinkClusterId()), 0,
+                "test" + SERVERS.PORT_0, true, pluginConfig, dstTestRuntime);
+        destContext.addSessionNameForLogging(session, "session_1");
```

Comment:
it will be good to declare "session_1" as a constant for the class to avoid hard-coding